packages/astro/src/core/errors/errors-data.ts

```diff
@@ -810,14 +803,14 @@ export const LocalImageUsedWrongly = {
 	name: 'LocalImageUsedWrongly',
 	title: 'Local images must be imported.',
 	message: (imageFilePath: string) =>
-		`\`Image\`'s and \`getImage\`'s \`src\` parameter must be an imported image or an URL, it cannot be a string filepath. Received \`${imageFilePath}\`.`,
-	hint: 'If you want to use an image from your `src` folder, you need to either import it or if the image is coming from a content collection, use the [image() schema helper](https://docs.astro.build/en/guides/images/#images-in-content-collections) See https://docs.astro.build/en/guides/images/#src-required for more information on the `src` property.',
```

Comment:
This hint was there because this is an insanely common mistake due to how unintuitive it is to not being able to pass image paths.

We've had a lot of support threads about it, which in my experience, being in the Discord all the time, decreased after we've improved this message by adding the hint.

Reply:
See longer response above: https://github.com/withastro/astro/pull/8833#discussion_r1367319517

I'm happy to work with you to improve any of these new hints, if we can keep my new breakdown of responsibility as "hints just hint, error reference URLs educate". I'm sure I err'd on the side of removing vs. keeping, so it makes total sense that some things should be added back.

Let me give this a rewrite and then LMK what you think.

Reply:
Done! LMK WYT.
  
<img width="704" alt="Screen Shot 2023-10-20 at 11 26 07 AM" src="https://github.com/withastro/astro/assets/622227/5047d45e-0dcb-4e5c-b52a-fbe45f02ac8c">

Reply:
I have to agree with Erika's original wording here, or something closer to that. This is not only how we phrase it in docs (you have to import the image first to be able to use it) but also what we keep saying over and over in support. And, that phrase does seem to get the idea across to people. I think the new suggestion loses that. ("use the ESM  import keyword" seems less helpful than "import the image")?